PyInstaller/hooks/hook-botocore.py

```diff
@@ -16,8 +16,14 @@
 #
 # https://botocore.readthedocs.org/en/latest/
 #
-# Tested with botocore 1.3.0
+# Tested with botocore 1.4.36
```

Comment:
I assume the hidden import only applies to version 4.x. Please check which version is the first one using this hidden import and use `is_module_satisfies('botocore >= 1.3.1')` to check for this version.